src/Nethermind/Nethermind.Abi/AbiSignature.cs

```diff
@@ -29,10 +32,12 @@ public AbiSignature(string name, params AbiType[] types)
 
         public string Name { get; }
         public AbiType[] Types { get; }
+        public byte[] Address => Keccak.Bytes.Slice(0, 4);
+        public Keccak Keccak => _keccak ??= Keccak.Compute(ToString());
```

Comment:
in other places in the code we always call it Hash and leave Keccak as a type only